### PR TITLE
Added fake data methods 

### PR DESCRIPTION
**Pull request by ehamp**
added fake data methods for resources, reviews, resource reviews, addresses

### CODE REVIEW

**Comment by ehamp** [comment]
@aharelick Here's our fake data, thanks for looking at it

**Comment by sandlerben** [comment]
Please clean commit history

**Comment by sandlerben** [comment]
wat

File: app/models/location.py [hunk 1]
```diff
@@ -72,3 +72,21 @@ class Address(db.Model):
...
+
+        fake = Faker()
+        from random import choice
+        for i in range(count):
+            a = Address(
+                name=fake.name(),
+                street_address=fake.street_address(),
+                city=fake.city(),
+                state=fake.state(),
+                zip_code=choice(ZIPCode.query.all())
```
Comment (aharelick) [inline app/models/location.py]: we probably want to store this as a variable (perhaps named `zip_codes`) then we can say `zip_code=choice(zip_codes)` so we don't have to query each time.

File: app/models/resource.py [hunk 2]
```diff
@@ -12,16 +14,40 @@ class Resource(db.Model):
...
+        from random import choice
+
+        fake = Faker()
+
+        for i in range(count):
+            r = Resource(
+                name=fake.name(),
+                description=fake.text(),
+                website=fake.url(),
+                address=choice(Address.query.all())
```
Comment (aharelick) [inline app/models/resource.py]: same thing as the zip code situation above, store this as a variable before the for loop

File: app/models/resource.py [hunk 3]
```diff
@@ -30,3 +56,21 @@ class ResourceReview(db.Model):
...
+        from faker import Faker
+
+        fake = Faker()
+
+        for i in range(count):
+            r = ResourceReview(
+                timestamp=fake.date_time(),
+                content=fake.text(),
+                rating=randint(1, 5),
+                count_likes=randint(1, 500),
```
Comment (aharelick) [inline app/models/resource.py:73]: damn, that's a lot of likes

**Comment by ehamp** [comment]
@aharelick  updated!

File: app/models/location.py [hunk 4]
```diff
@@ -72,3 +72,23 @@ class Address(db.Model):
 
     def __repr__(self):
         return '<Address \'%s\'>' % self.name
+
+    @staticmethod
+    def generate_fake(count=10):
+        """Generate count fake Resources for testing."""
```
Comment (aharelick) [inline app/models/location.py]: `"""Generate count fake Addresses for testing."""`

**Comment by aharelick** [comment]
@ehamp, please fix that one comment on the comment, then ping me

**Comment by ehamp** [comment]
@aharelick fixed

**Comment by aharelick** [comment]
@ehamp, yeah go for it.
merge
^^ These are two arrows merging.

**Pull request merged**